packages/cardano-services/src/StakePool/DbSyncStakePoolProvider/util.ts

```diff
@@ -42,3 +44,8 @@ export const emptyPoolsExtraInfo = {
   poolRetirements: [],
   poolRewards: []
 };
+
+export const toHexString = (bytes: Buffer) => bytes.toString('hex');
```

Comment:
This belongs in [src/util/hexString](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-services/src/util/hexString.ts)

Reply:
Done in aa0a867